Modules/Devices/test/Devices.Domain.Tests/Identities/StartDeletionProcessBySupportTests.cs

```diff
@@ -0,0 +1,80 @@
+﻿using Backbone.BuildingBlocks.Domain;
+using Backbone.DevelopmentKit.Identity.ValueObjects;
+using Backbone.Modules.Devices.Domain.Aggregates.Tier;
+using Backbone.Modules.Devices.Domain.Entities.Identities;
+using Backbone.Modules.Devices.Domain.Tests.Identities.TestDoubles;
+using Backbone.Tooling;
+using FluentAssertions;
+using Xunit;
+
+namespace Backbone.Modules.Devices.Domain.Tests.Identities;
+
+public class StartDeletionProcessBySupportTests
+{
+    [Fact]
+    public void Start_deletion_process_as_support()
+    {
+        // Arrange
+        SystemTime.Set(DateTime.Parse("2000-01-01"));
+        var activeIdentity = CreateIdentity();
+
+        Hasher.SetHasher(new DummyHasher(new byte[] { 1, 2, 3 }));
```

Comment:
Why do you set the hasher?

Reply:
The Hasher is validated and it throws the following if not set:
```
Expected auditLogEntry.IdentityAddressHash to be a collection with 3 item(s), but {0x1C, 0xF4, 0x94, 0x2D, 0x39, 0x7D, 0xD3, 0x2F, 0xE1, 0xEC, 0xA5, 0x26, 0x78, 0x59, 0x00, 0x23, 0xB2, 0x9F, 0x08, 0x11, 0x9B, 0x1A, 0x61, 0x2A, 0x70, 0x34, 0x77, 0xB7, 0x10, 0xD6, 0xE1, 0x80}
contains 29 item(s) more than
{0x01, 0x02, 0x03}.
```

Reply:
True. My bad, sorry